src/ring_theory/ideal_operations.lean

```diff
@@ -577,22 +577,19 @@ by rw [←submodule.ext'_iff, ker_eq]; exact is_add_group_hom.inj_iff_trivial_ke
 lemma ker_eq_bot_iff_eq_zero : ker f = ⊥ ↔ ∀ x, f x = 0 → x = 0 :=
 by rw [←submodule.ext'_iff, ker_eq]; exact is_add_group_hom.trivial_ker_iff_eq_zero f
 
-lemma injective_iff : function.injective f ↔ ∀ x, f x = 0 → x = 0 :=
```

Comment:
Why did you remove this one? Is it now duplicate?

Reply:
Yes, see `algebra/ring`.